pkg/webhook/inject/inject_test.go

```diff
@@ -24,17 +26,23 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
 	controllerCfg "github.com/chaos-mesh/chaos-mesh/pkg/config"
+	"github.com/chaos-mesh/chaos-mesh/pkg/metrics"
 	"github.com/chaos-mesh/chaos-mesh/pkg/webhook/config"
 )
 
 var _ = Describe("webhook inject", func() {
+	var i Injector
 
 	Context("Inject", func() {
 		It("should return unexpected end of JSON input", func() {
 			var testClient client.Client
 			var cfg *config.Config
 			var controllerCfg *controllerCfg.ChaosControllerConfig
-			res := Inject(&admissionv1.AdmissionRequest{}, testClient, cfg, controllerCfg, nil)
+			var metrics *metrics.ChaosControllerManagerMetricsCollector
+			var log logr.Logger
```

Comment:
please initialize this `log`, you could take a look on here as reference:

https://github.com/chaos-mesh/chaos-mesh/blob/master/cmd/chaos-controller-manager/provider/suit_test.go#L81-L82